.github/CODEOWNERS

```diff
@@ -37,6 +37,7 @@ src/runtime_src/xocl/api/plugin/xdp/ @jvillarre
 src/runtime_src/xdp/ @jvillarre
 src/runtime_src/doc/ @uday610
 src/runtime_src/tools/ @rozumx
+src/runtime_src/tools/xclbinutil/ @feirui
```

Comment:
Hi Fei , i see your username is @xfreid .  you added @feirui here. i am not sure whether this works or not.

Reply:
This will not work.  See the underlined error.

Reply:
My bad, just fixed it.